uint64                    human_id
geometry_msgs/PoseStamped pose
